src/Decompress.php

```diff
@@ -29,10 +29,10 @@ public function __invoke($value)
      *
      * Decompresses the content $value with the defined settings
      *
-     * @param  string $value Content to decompress
-     * @return string The decompressed content
+     * @param mixed $value Content to decompress
+     * @return mixed The decompressed content
      */
-    public function filter($value)
+    public function filter(mixed $value): mixed
     {
         if (! is_string($value) && $value !== null) {
```

Comment:
wait, we decompress `null`? :thinking:

Reply:
I'll have to look closer at this later on, but I think that some of the adapters only work on files passed as options (Those adapters might be dead now), therefore there's no argument to `filter()`, or something like that…